src/OrganisationRegistry/Organisation/Organisation.cs

```diff
@@ -1219,16 +1220,16 @@ public void UpdateKboFormalNameLabel(IMagdaName kboFormalName, LabelType formalN
                 location.FormattedAddress,
                 false,
                 locationType?.Id,
-                locationType?.Name,
+                locationType?.Name ?? "",
                 validity.Start,
                 validity.End));
         }
 
         public void UpdateKboRegisteredOfficeLocations(
-            KboRegisteredOffice newKboRegisteredOffice,
```

Comment:
I think it would be more fitting to have this method not be called, or return immediately when KboRegisteredOffice is null